waiter/integration/waiter/token_request_test.clj

```diff
@@ -657,7 +657,7 @@
                   (reset! cookies-atom cookies)
                   (is (= "Added cookie x-waiter-consent" body))
                   ; x-waiter-consent should be emitted Waiter
-                  (is (contains? (set (map :name cookies)) "x-waiter-consent"))
+                  (is (some #(= (:name %) "x-waiter-consent") cookies))
```

Comment:
Nitpick, can we extract `#(= (:name %) "x-waiter-consent")` into a variable or helper function?

Reply:
Done.